src/output-filestore.c

```diff
@@ -492,9 +493,15 @@ static OutputInitResult OutputFilestoreLogInitCtx(ConfNode *conf)
                        "from conf file - %s.  Killing engine",
                        stream_depth_str);
             exit(EXIT_FAILURE);
-        } else {
-            FileReassemblyDepthEnable(stream_depth);
         }
+        if (stream_depth && stream_depth <= stream_config.reassembly_depth) {
+            SCLogWarning(SC_WARN_FILESTORE_CONFIG, "file-store.stream-depth value "
+                       "%"PRIu32" must be larger than stream.reassembly.depth %"PRIu32"."
+                       " Using %"PRIu32" instead",
+                       stream_depth, stream_config.reassembly_depth, stream_config.reassembly_depth);
```

Comment:
I'm not keen on the wording here.  If it *must* be larger than stream re-assembly, we should fail here. But its really more of a "should", as its basically has no affect if set to less than the stream reassembly depth.

Reply:
Thanks so something like a warning with this message "file-store-stream-depth has no effect since it's less than stream.reassembly-depth"?

Reply:
Yeah, IMO, thats better.